ssz/src/decode.rs

```diff
@@ -132,7 +132,7 @@ pub struct Offset {
 ///
 /// See [`SszDecoder`](struct.SszDecoder.html) for usage examples.
 pub struct SszDecoderBuilder<'a> {
-    bytes: &'a [u8],
+    __bytes: &'a [u8],
```

Comment:
I don't think we should modify the name of the struct field (it's not necessary).